src/clusterfuzz/_internal/tests/core/datastore/data_handler_test.py

```diff
@@ -1025,3 +1025,91 @@ def test_fuzzer_created(self):
         self.job.name, None, None, None, None, None, None, None,
         timeout_multiplier, None, True)
     self.assertTrue(data_handler.get_testcase_by_id(testcase_id).trusted)
+
+
+@test_utils.with_cloud_emulators('datastore')
+class GetEntitiesTest(unittest.TestCase):
+  """Tests for get_entities."""
+
+  class GetEntitiesTestModel(data_types.Model):
+    """Model for get_entities tests."""
+    name = ndb.StringProperty()
+    value = ndb.IntegerProperty()
+    timestamp = ndb.DateTimeProperty()
+
+  def setUp(self):
+    self.entity1 = self.GetEntitiesTestModel(
+        name='a', value=1, timestamp=datetime.datetime(2023, 1, 1, 0, 0, 1))
+    self.entity1.put()
+    self.entity2 = self.GetEntitiesTestModel(
+        name='b', value=2, timestamp=datetime.datetime(2023, 1, 1, 0, 0, 2))
+    self.entity2.put()
+    self.entity3 = self.GetEntitiesTestModel(
+        name='c', value=1, timestamp=datetime.datetime(2023, 1, 1, 0, 0, 3))
+    self.entity3.put()
+    self.entity4 = self.GetEntitiesTestModel(
+        name='d', value=3, timestamp=datetime.datetime(2023, 1, 1, 0, 0, 4))
+    self.entity4.put()
+
+  def test_get_all(self):
+    """Test getting all entities."""
+    result = data_handler.get_entities(self.GetEntitiesTestModel)
+    self.assertEqual(4, len(result))
+
+  def test_with_limit(self):
+    """Test with a limit."""
```

Comment:
Docstrings that do not provide any new information should not be used.
go/pystyle#test-docs

This might also be useful go/python-readability-advice#test_docstrings